ui/src/app/pages/admin/admin-user.tsx

```diff
@@ -124,6 +112,36 @@ const EmailValidationErrorMessage = ({emailValidationResponse, updatedProfile, v
   return null;
 };
 
+interface FreeCreditsProps {
+  isAboveLimit: boolean;
+  usage: string;
+}
+
+const FreeCreditsUsage = ({isAboveLimit, usage}: FreeCreditsProps) => {
+  const inputStyle = isAboveLimit ?
+  {...styles.textInput,
+    backgroundColor: colorWithWhiteness(colors.danger, .95),
+    borderColor: colors.danger,
+    color: colors.danger,
+  } :
+  {...styles.textInput,
+    ...styles.backgroundColorDark,
+    color: colors.disabled,
+  };
+
+  return <React.Fragment>
+    <TextInputWithLabel
+      labelText={'Free credits used'}
```

Comment:
nit: don't think you need the curly braces for a string literal with no interpolation.

Reply:
yep